examples/reference/widgets/Tabulator.ipynb

```diff
@@ -19,7 +19,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The `Tabulator` widget allows displaying and editing a pandas DataFrame. The `Tabulator` is a largely backward compatible replacement for the [`DataFrame`](./DataFrame.ipynb) widget and will eventually replace it. It is built on the **version 5.5** of the [Tabulator](http://tabulator.info/) library, which provides for a wide range of features.\n",
+    "The `Tabulator` widget allows displaying and editing a pandas DataFrame. The `Tabulator` is a largely backward compatible replacement for the [`DataFrame`](./DataFrame.ipynb) widget and will eventually replace it. It is built on the **version 6.2.1** of the [Tabulator](http://tabulator.info/) library, which provides for a wide range of features.\n",
```

Comment:
```suggestion
    "The `Tabulator` widget allows displaying and editing a pandas DataFrame. The `Tabulator` is a largely backward compatible replacement for the [`DataFrame`](./DataFrame.ipynb) widget and will eventually replace it. It is built on the **version {{TABULATOR_VERSION}}** of the [Tabulator](http://tabulator.info/) library, which provides for a wide range of features.\n",
```

Reply:
Ok by doing that the .ipynb will take the version number from conf.py? But the correct version number will then still have to be entered in conf.py manually? Or can it be automated there as well?

Reply:
Yes, you should be able to get the version automatically with `from panel.models.tabulator import TABULATOR_VERSION`